staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -157,6 +174,11 @@ func handleRootHealthz(checks ...HealthzChecker) http.HandlerFunc {
 				fmt.Fprintf(&verboseOut, "[+]%v ok\n", check.Name())
 			}
 		}
+		if excluded.Len() > 0 {
+			fmt.Fprintf(&verboseOut, "err: some health checks cannot be excluded: no matches for %v\n", formatQuoted(excluded.List()...))
```

Comment:
Sorry to be nitpicky, but I'd call this a warning and not an error, since it doesn't make the check fail and the word "error" makes admins nervous :)

Reply:
Sure, I can remove the prefix. I added it after the fact because it made it slightly easier to parse the string out from the verbose output blob.